docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1327,23 +1327,65 @@ To enable export to parquet, an administrator must:
+|`id`                                                  |Globally unique identifier for the audit event.|
+|`outcome`                                             |Action outcome. Possible values are "success", "failure", "unknown", and "pending".|
+|`typeURI`                                             |TypeURI property of the CADF event entity. Value is always "http://schemas.dmtf.org/cloud/audit/1.0/event".|
+|`attachments`                                         |Note: Contains FHIR server-specific audit event data.|
+|`attachments:contentType`                             |FHIR server-specific audit event data content type. Value is always "application/json".|
+|`attachments:content`                                 |Note: Contents of this field (with its subfields) is encoded as Base64.
+|`attachments:content:request_unique_id`               |Globally unique identifier for the FHIR server request.|
+|`attachments:content:action`                          |FHIR action type. Possible values are "C" (create), "U" (update), "R" (read), "D" (delete), "P" (patch), and "O" (custom operation).|
+|`attachments:content:operation_name`                  |FHIR custom operation name.|
+|`attachments:content:start_time`                      |FHIR request start time.|
+|`attachments:content:end_time`                        |FHIR request end time.|
+|`attachments:content:api_parameters:request`          |FHIR request URL.|
+|`attachments:content:api_parameters:request_status`   |FHIR request HTTP status (e.g. 200).|
+|`attachments:content:data:resource_type`              |Resource type of FHIR resource that was created, updated, or deleted.|
+|`attachments:content:data:id`                         |Resource ID of FHIR resource that was created, updated, or deleted.|
+|`attachments:content:data:version`                    |Updated version of FHIR resource that was created, updated, or deleted.|
+|`attachments:content:batch:resources_read`            |FHIR resource count retrieved on a search request.|
+|`attachments:content:event_type`                      |FHIR event type. Possible values are "fhir-create", "fhir-update", "fhir-patch", "fhir-delete", "fhir-read", "fhir-version-read", "fhir-history", "fhir-search", "fhir-bundle", "fhir-validate", "fhir-metadata", "fhir-configdata", and "fhir-operation".|
+|`attachments:content:description`                     |FHIR event description.|
+|`attachments:content:client_cert_cn`                  |Value is determined by "IBM-App-cli-CN" HTTP header of the FHIR request.|
+|`attachments:content:client_cert_issuer_ou`           |Value is determined by "IBM-App-iss-OU" HTTP header of the FHIR request.|
+|`attachments:content:location`                        |IP address and hostname of the source of the FHIR request.|
+|`initiator:id`                                        |Value is always "TENANT_ID@fhir-server", where TENANT_ID is replaced with the tenant ID.|
```

Comment:
this one is particularly interesting for me...we'll need to think about how to get this populated properly for deployments where we have a server-per-tenant and they all use tenant.  or maybe its fine to be `default` in those cases (and you need to check the `attachments:content:api_parameters:request` or `initiator:host` to determine the tenant for those ones)